grouper/fe/handlers/group_remove.py

```diff
@@ -41,8 +44,40 @@ def post(self, group_id=None, name=None):
                 reason="Can't remove a service account user from the service account group."
             )
 
-        group.revoke_member(self.current_user, removed_member, "Removed by owner/np-owner/manager")
-        AuditLog.log(self.session, self.current_user.id, 'remove_from_group',
-                     '{} was removed from the group.'.format(removed_member.name),
-                     on_group_id=group.id, on_user_id=removed_member.id)
+        try:
+            group.revoke_member(
+                self.current_user,
+                removed_member,
+                "Removed by owner/np-owner/manager"
+            )
+
+            AuditLog.log(self.session, self.current_user.id, 'remove_from_group',
+                         '{} was removed from the group.'.format(removed_member.name),
+                         on_group_id=group.id, on_user_id=removed_member.id)
+        except Exception as e:
+            alert = Alert("danger", str(e))
+            return self._render_group_with_alert(group, alert)
+
         return self.redirect("/groups/{}?refresh=yes".format(group.name))
+
+    def _render_group_with_alert(self, group, alert):
```

Comment:
I would like to replace this with a redirect that preserves the alerts, a la Rails' flash, to reduce code duplication.